components/21-atoms/accolade/_accolade.scss

```diff
@@ -201,14 +203,14 @@ $accolade-height: 1rem;
   &.left-top,
   &.left-center,
   &.left-bottom {
-    left: -($accolade-height/2);
+    left: -($accolade-height*.5);
     transform: rotate(90deg);
   }
 
   &.right-top,
   &.right-center,
   &.right-bottom {
-    right: -($accolade-height/2);
+    right: -($accolade-height*.5);
```

Comment:
Add space between * and .5. Makes it clearer that it is a calculation instead of a wildcard.

Reply:
<img width="815" alt="image" src="https://user-images.githubusercontent.com/1823998/219341851-de6c0a9c-9d8b-434d-afe1-78843a4b55d4.png">

Al deze lijnen zijn output van het script `sass-migrator division **/*.scss`.
Blijkbaar stonden er vroeger soms wel en soms geen spaties, als we dat opkuisen kunnen we dat beter in een aparte PR doen (los van de node migratie).

Ps: met de GitHub suggesties knop kan je de wijziging meteen in de code toepassen:
```suggestion
    right: -($accolade-height * .5);
```